src/main/java/dk/kb/labsapi/api/impl/LabsapiService.java

```diff
@@ -206,6 +206,10 @@ public StreamingOutput exportFields(String query, List<String> fields, Long max,
                 httpServletResponse.setContentType( "application/x-ndjson");
                 break;
             }
+            case txt: {
+                httpServletResponse.setContentType("text/plain");
```

Comment:
Maybe specify that it is UTF-8? `text/plain;charset=UTF-8`.

It is implicit for the two JSON variants, but while you are at it, please add the `;charset=UTF-8` to CSV as well